sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    !curl https://certs.secureserver.net/repository/sf-class2-root.crt -O 
</pre><p>Why is this a trusted source? Can the certificate be retrieved by a source from within AWS rather than the public internet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thank you for comment.   According to AWS documentation this is an URL should be used to get for certificate  --  https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_keyspaces.html</p>